### ISSUE TITLE
eliminate "about us"

### ISSUE THREAD

**Issue by jberkus** [vmbrasseur/10stepprogram#29]
... since "don't have an About Me" is one of my general pieces of advice.  And it's not like people don't know who we are.

**Comment by vmbrasseur** [comment]
I disagree with "have no 'about me' at all." I agree with "minimal 'about me', no more than ~20 seconds."

**Comment by jberkus** [comment]
Heh, seems like we have a difference of opinion about something.  Maybe we should have no About Josh so that we can explain that.  ;-)
